com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.Prepass.cs

```diff
@@ -280,19 +288,28 @@ void BindMotionVectorPassColorBuffers(in RenderGraphBuilder builder, in PrepassO
             return result;
         }
 
+        class UpdateColorMaskForUnlitPassData
+        {
+            public ColorWriteMask colorWriteMask;
+        }
+
+        void UpdateColorMaskForUnlit(RenderGraph renderGraph, HDCamera hdCamera)
+        {
+            using (var builder = renderGraph.AddRenderPass<UpdateColorMaskForUnlitPassData>("UpdateColorMaskForUnlit", out var passData))
+            {
+                // Disable write to normal buffer for unlit shader (the normal buffer binding change when using MSAA)
```

Comment:
normal write buffer for unlit is enabled when unlit is a shadow matte (only supported with shader graph), so this comment is wrong.